handlers/tokeninfo/jwt/handler.go

```diff
@@ -16,21 +18,26 @@ type jwtHandler struct {
 }
 
 func (h *jwtHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	start := time.Now()
 	ti, err := h.validateToken(r)
 	if err != nil {
+		var tie tokeninfo.TokenInfoError
 		switch err {
 		case jwt.ErrNoTokenInRequest:
-			tokeninfo.Error(w, tokeninfo.ErrInvalidRequest)
+			tie = tokeninfo.ErrInvalidRequest
 		default:
-			tokeninfo.Error(w, tokeninfo.ErrInvalidToken)
+			tie = tokeninfo.ErrInvalidToken
 		}
-		log.Debug(err)
+		registerError(tie)
+		tokeninfo.Error(w, tie)
+		log.Println(err)
 		return
 	}
 
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(http.StatusOK)
 	json.NewEncoder(w).Encode(ti)
+	measureRequest(start, fmt.Sprintf("planb.tokeinfo.jwt.%s.requests", ti.Realm))
```

Comment:
this crashes with nil pointer when the result from h.validateToken(r) is nil, nil